### PR TITLE
Ekr copy edit

### PR DESCRIPTION
**Pull request by ekr**
This is my copy-edit pass, combined with some of Ben's comments.

### CODE REVIEW

File: rfc8446.xml [hunk 1]
```diff
@@ -826,7 +826,7 @@ at runtime is not prescribed by the presentation language.</t>
 
 <t>The handshake protocol is used to negotiate the security parameters
 of a connection. Handshake messages are supplied to the TLS record layer, where
-they are encapsulated within one or more TLSPlaintext or TLSCiphertext structures, which are
+they are encapsulated within one or more TLSPlaintext or TLSCiphertext structure  which are
```
Comment (kaduk) [inline rfc8446.xml:829]: I think there's a rule for whether to use singular or plural in these cases, but  I forget what the rule says

File: rfc8446.xml [hunk 2]
```diff
@@ -1148,7 +1153,8 @@ negotiate the protocol version. All TLS&nbsp;1.3 ServerHello messages MUST
 contain the "supported_versions" extension.  Current ServerHello messages
 additionally contain either the "pre_shared_key" extension or the "key_share"
 extension, or both, when using a PSK with (EC)DHE key establishment. Other
-extensions are sent separately in the EncryptedExtensions message.</t>
+extensions  (see  <xref target="extensions"/>)
```
Comment (kaduk) [inline rfc8446.xml:1156]: I don't *think* the extra space here makes it into the rendered text...

File: rfc8446.xml [hunk 3]
```diff
@@ -2644,7 +2651,7 @@ containing a single certificate and set of extensions.</t>
   A set of extension values for the CertificateEntry. The "Extension"
 format is defined in <xref target="extensions"/>. Valid extensions for server
 certificates at present include the OCSP Status extension <xref
-target="RFC6066"/> and SignedCertificateTimestamp <xref target="RFC6962"/>;
+target="RFC6066"/> and SignedCertificateTimestamp extension <xref target="RFC6962"/>;
```
Comment (kaduk) [inline rfc8446.xml:2654]: Do we also need to add "the" for proper parallelism?

File: rfc8446.xml [hunk 4]
```diff
@@ -4175,7 +4183,7 @@ the server system as a whole.</t>
 
 <t>The first class of attack can be prevented by sharing state to guarantee that
 the 0-RTT data is accepted at most once.  Servers SHOULD provide that level of
-replay safety, by implementing one of the methods described in this section or
+replay safety by implementing one of the methods described in this section or
```
Comment (kaduk) [inline rfc8446.xml:4186]: If I was author I would keep this comma, but I don't think my opinion is actually relevant in this case.

File: rfc8446.xml [hunk 5]
```diff
@@ -159,10 +159,10 @@ are considered legacy. Those that remain all use Authenticated Encryption
 with Associated Data (AEAD) algorithms. The cipher suite concept has been
 changed to separate the authentication and key exchange mechanisms from
 the record protection algorithm (including secret key length) and a hash
-to be used with the key derivation function and the Hashed Message
-Authentication Code (HMAC).</t>
+to be used with both the key derivation function and handshake
+message authentication code.</t>
```
Comment (kaduk) [inline rfc8446.xml:163]: While this certainly flows more nicely, isn't this hash also used (via HMAC/HKDF) in the key hierarchy?

Comment (kaduk) [reply rfc8446.xml:163]: (Also, if we take out the HMAC expansion here, don't we end up having to expand it as part of HKDF a couple paragraphs down, which would be pretty ugly?)

File: rfc8446.xml (hunk 2 above)
Comment (ekr) [reply rfc8446.xml:1156]: I certainly hope not.

File: rfc8446.xml (hunk 3 above)
Comment (ekr) [reply rfc8446.xml:2654]: I can do that.

File: rfc8446.xml (hunk 5 above)
Comment (ekr) [reply rfc8446.xml:163]: I would actually prefer to expand neither of these, just as we don't expand AES.

**Pull request closed without merging**